mw-webapp/cypress/accessIds/allWaysAccessIds.ts

```diff
@@ -0,0 +1,6 @@
+export const allWaysAccessIds = {
+    titles: {
+        allWays : 'ways-title',

```

Comment:
Is it possible to use camelCase instead of kebab-case for consistancy. If yes, please  check everywhere

Reply:
Ok